addons/website_event_track_quiz/controllers/main.py

```diff
@@ -0,0 +1,144 @@
+import json
+
+from odoo import http
+from odoo.addons.website_event_track_session.controllers.session import WebsiteEventSessionController
+from odoo.exceptions import AccessError, UserError
+from odoo.http import request
+
+
+class WebsiteEventTrackQuiz(WebsiteEventSessionController):
+    def _fetch_track(self, track_id):
```

Comment:
Could probably be improved (see other comments) with an event_id, and check that
 * user can read event
 * user can read track
 * event has started (unless belongs to event group)